requirements_dev.txt

```diff
@@ -0,0 +1,2 @@
+pytest==7.1.3
```

Comment:
Use poetry's dev dependencies: `poetry add -D`

Reply:
I used it for `tox`, but I found a different approach to use poetry's dev deps.